packages/service-library/tests/test_redis.py

```diff
@@ -262,7 +263,9 @@ async def _inc_counter() -> None:
             ):
                 await counter.race_condition_increase(INCREASE_BY)
 
-    await asyncio.gather(*(_inc_counter() for _ in range(INCREASE_OPERATIONS)))
+    await logged_gather(
+        *(_inc_counter() for _ in range(INCREASE_OPERATIONS)), max_concurrency=15
+    )
     assert counter.value == INCREASE_BY * INCREASE_OPERATIONS
```

Comment:
Please start using ```limited_gather``` instead of ```logged_gather``` I think is a better implementation. It does not use a semaphore, and creates coroutine as needed not all of them in one go and then 15 run and all the rest blocks.
there: https://github.com/ITISFoundation/osparc-simcore/blob/46e3c9bf021106850d5353f775fd7aff083047a9/packages/service-library/src/servicelib/utils.py